src/main/java/org/intellij/sdk/codesync/clients/CodeSyncClient.java

```diff
@@ -47,18 +47,22 @@ public Boolean isServerUp() {
 
     @return  Pair<Boolean, User>  First item of the pair shows of token is valid and secod is the user instance.
      */
-    public Pair<Boolean, User> getUser(String accessToken) throws RequestError {
+    public Pair<Boolean, User> getUser(String repoPath, String accessToken) throws RequestError {
```

Comment:
I think we do not need changes in this method, these seem redundant. Can you remove these changes and test the flow to make sure.
 
Also, just a note on coding good practices, this is method is getting user details using access token and it does not make sense to have `repoPath` in its arguments since user's details are not dependent on the repo 🙂.